.github/workflows/generate-tests.yaml

```diff
@@ -0,0 +1,28 @@
+name: Generate tests for new kind-node image
+
+on: 
+  repository_dispatch:
+    types: [generate-tests]           
+
+jobs:
+  test:
+    name: Generate test files
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.ACTION_TOKEN }}
+      - name: Trim version
+        run: echo "LATEST_KIND_NODE=$(echo ${{ github.event.client_payload.latest-kind-node }} | tr -d '[:v]')" >> $GITHUB_ENV
+      - name: Generate and commit tests
```

Comment:
Does this step commits the test result?

Reply:
No, good spot. Will change the step `name`